public/app/features/dashboard/state/initDashboard.ts

```diff
@@ -102,6 +103,29 @@ async function fetchDashboard(
   }
 }
 
+const logQueryOnDashboardPanels = (panels: PanelModel[], dashboardId: string, collapsed?: boolean): void => {
+  panels.forEach((panel) => {
+    // panel is collapsed or parent is
+    collapsed = !!panel.collapsed || !!collapsed;
+    if (panel.panels) {
+      logQueryOnDashboardPanels(panel.panels, dashboardId, collapsed);
+    } else {
+      panel.targets.forEach((target) => {
+        reportInteraction('grafana_dashboard_loaded', {
```

Comment:
The event name would imply this event is only triggered once per dashboard, but in practice it can be executed thousands of times when a dashboard is loaded.

Reply:
Link of discussion for reference: https://github.com/grafana/grafana/pull/50656#discussion_r915772275